config/dev.exs

```diff
@@ -23,8 +23,9 @@ config :malan, MalanWeb.Endpoint,
   # http: [ip: {127, 0, 0, 1}, port: 4000],
   check_origin: false,
   code_reloader: true,
-  debug_errors: true,
-  # debug_errors: false,
+  # Set `debug_errors: true` to get more useful output for any exceptions
+  # debug_errors: true,
+  debug_errors: false,
```

Comment:
maybe this could include an env var check so that anyone working on malan doesn't have to deal with local changes that must not be committed?

while i'm here, i'm also noticing `code_reloader: true`. i saw some log output from my docker container about how it can't reload code because of a missing dependency. i'm sure it's useful when working on malan, but it's not something i'd miss when integrating malan into a dev environment.

Reply:
@cainlevy great minds think alike!  that was my first preference too, but I maddeningly I couldn't get it to work.  After some research, I discovered that `debug_errors` is actually compiled in, so it can't be changed at runtime (only at compile time).  I even set an env var for compile time but the compiler doesn't recognize that it changes anything so doesn't recompile the appropriate parts.  Unfortunately I don't think it's worth the effort required (at least right now) to figure out how to make it work